text/0000-issue-workflow.md

```diff
@@ -0,0 +1,222 @@
+        - Comment and close
+    - Maybe - needs RFC
+        - Remove label `status/triage`
+        - Add label `status/requires-rfc`
+        - Comment and close
+    - Maybe - needs alignment
+        - Remove label `status/triage`
+        - Add label `status/discussion-needed`
+        - Add appropriate `type/*`
+    - Maybe - I can’t tell from the information in the issue
+        - Remove label `status/triage`
+        - Add label `status/incomplete`
+        - Request further information from issue author
+        - If activity is missing for 7 business days, follow up or close
+    - Yes - but it’s not on our roadmap (would consider a PR)
+        - Remove label `status/triage`
+        - Add label `status/not-planned`
+        - Comment and close
+    - Yes
+        - Could this be worked on right now?
+            - Yes
+                - Remove label `status/triage`
+                - Add label `status/ready`
+                - Add appropriate `size/*` (if possible) and `type/*`
+            - No - needs clearer definition of done
+                - Remove label `status/triage`
+                - Add acceptance and label `status/incomplete`
+                - Add appropriate `size/*` (if possible) and `type/*`
+                - If activity is missing for 7 business days, follow up or close
+            - No - needs alignment
+                - Remove label `status/triage`
+                - Add label `status/discussion-needed`
+                - Add appropriate `size/*` (if possible) and `type/*`
+            - No - blocked by other work
+                - Remove label `status/triage`
+                - Add label `status/blocked`
+                - Add appropriate `size/*` (if possible) and `type/*`
+
+#### Pull Requests
+
+> Ignoring:
+> - Drafts
+> - PRs that have recent activity
+
+- As a contributor...
+    - If no one has responded within (2 business days):
+        - Notify appropriate maintainers
+    - If author has not updated PR after feedback within (30 days):
+        - Follow up\
+          OR
+        - Close
+- As a maintainer...
+    - Does code look good?
+        - Yes
+            - Remove `status/triage`
+            - Add `status/code-approved`
+            - Did user acceptance pass?
+                - Yes
+                    - Add `status/accepted`
+                    - Merge
+                - No
+                    - Add `status/incomplete`
+                    - Submit review with changes requested
+        - No - needs some changes
+            - Remove `status/triage`
+            - Add `status/incomplete`
+            - Submit review with changes requested
+### Automation
+
+This RFC does not call any specific automation but in providing processes around issues and pull requests
+it would enable further automation such as:
+
+- Labels can be synchronized across repositories ([RFC#53](https://github.com/buildpacks/rfcs/pull/53)).
+- Block PR merges until `status/accepted` label is added.
```

Comment:
Given PR merge can be blocked pending code owner approval, this automation seems redundant